README.md

```diff
@@ -7,20 +7,20 @@
 Wrapper for fetch to simplify sending and receiving RDF data.
 
 This is the light version of the `@rdfjs/fetch` package without the `@rdfjs/formats-common` dependency.
-It can be usefully if you want to make a build for the browser with a reduced set of parsers and serializers.
+It is useful when you want to make a build for the browser with a reduced set of parsers and serializers.
 The `formats` options is required for this package.
 See also the `@rdfjs/fetch` documentation.
 
 ## Usage
 
-The package exports a fetch function which wraps the request and response object for on the fly RDF quad processing.
-The function accepts the same parameters like [fetch](https://fetch.spec.whatwg.org/), but also accepts some addition options and provides additional methods.
+The package exports a fetch function which wraps the request and response object to process RDF quads on the fly.
+The function accepts the same parameters as [fetch](https://fetch.spec.whatwg.org/) and some additional options. It also provides additional methods.
 
 ### Options
 
 The `options` object accepts the following additional parameters:
 
-- `formats`: An [formats-common](https://github.com/rdfjs/formats-common) compatible object which contains a set of parsers and serializers.
+- `formats`: A [formats-common](https://github.com/rdfjs/formats-common) compatible object which contains a set of parsers and serializers.
```

Comment:
An example would be good here, it's not clear what a formats-common compatible object is.